src/commands/env/compute/collaborator/add.ts

```diff
@@ -64,7 +62,7 @@ export default class ComputeCollaboratorAdd extends Command {
       }
 
       if (error.message?.includes('404')) {
-        this.error(`${herokuColor.heroku(herokuUser)} does not exist.`);
+        this.error(`Couldn't find Heroku user ${herokuColor.heroku(herokuUser)}.`);
```

Comment:
Change in verbiage to that specified in requirements doc

Reply:
He didn't mention this change in our meeting so I would confirm that update with Shyam in the PRD. He may not have pulled it from the current output.

Reply:
He confirmed that the text in the doc was from old code, so I will revert that line.